src/main/resources/db/migration/V1_3__add_token_blacklist_sequence.sql

```diff
@@ -0,0 +1,3 @@
+CREATE SEQUENCE token_blacklist_seq
```

Comment:
This has already been done as part of a migration to fix the previous migration that I broke so suspect this won't work when merged into develop